packages/nimble-components/src/table-column/number-text/index.ts

```diff
@@ -111,7 +130,12 @@ export class TableColumnNumberText extends TableColumnTextBase {
             case NumberTextFormat.decimal:
                 return new DecimalFormatter(
                     lang.getValueFor(this),
-                    this.decimalDigits ?? defaultDecimalDigits
+                    typeof this.decimalMaximumDigits === 'number'
+                        ? 0
+                        : this.decimalDigits ?? defaultDecimalDigits,
+                    this.decimalMaximumDigits
+                        ?? this.decimalDigits
+                        ?? defaultDecimalDigits
```

Comment:
Maybe move the logic for these last two parameters into some consts above the `return` statement?